src/wp-admin/edit-form-blocks.php

```diff
@@ -106,6 +106,24 @@ static function ( $classes ) {
 	'wp.blocks.unstable__bootstrapServerSideBlockDefinitions(' . wp_json_encode( get_block_editor_server_block_settings() ) . ');'
 );
 
+// Preload server-registered block bindings sources.
```

Comment:
Should we add here a `since 6.7` or that only applies to filters?

Reply:
I don't know, to be honest. I checked and the other places in that file where we are calling `wp_add_inline_script` don't have a `since` tag.

Reply:
Filters need `@since` because that information is extracted when generating the documentation. It isn't as helpful here, so it can be omitted.